src/galaxy/app.py

```diff
@@ -1324,3 +1325,15 @@ def __call__(self, bytes_amount):
             percentage = (self._seen_so_far / self._size) * 100
             logging.debug("\r%s  %s / %s  (%.2f%%)", self._filename,
                           self._seen_so_far, self._size, percentage)
+
+
+class SystemHealth(object):
+    @staticmethod
+    def monitor_endpoint(endpoint, request_type, body=None):
+        """returns the response status code for a request made to a particular API endpoint"""
+        if request_type == 'GET':
+            get_req = requests.get(endpoint)
+            return get_req.status_code
+        if request_type == 'POST':
+            post_req = requests.post(endpoint, json=body, headers={'access-token':ACCESS_TOKEN})
```

Comment:
Just a question , I can see from the website that access_token is not going to expire but have you tried it ? Do they remain  same all the time ? Since you are reading it from config , I am just concerned if they will expire or not . Generally access token do expire